LibraBFT/Impl/Handle/Properties.agda

```diff
@@ -51,6 +51,7 @@ initRMSatisfiesInv =
     (mkSafetyRulesInv (mkSafetyDataInv refl z≤n))
 
 invariantsCorrect -- TODO-1: Decide whether this and direct corollaries should live in an `Properties.Invariants` module
+                  -- TODO-2: This needs to be tightend so that the msgPool is of the previous system state (if there is one)
```

Comment:
I don't think this is right or makes sense.  This is an invariant (as opposed to a transition property), so what previous state?  I think we will be *applying* the invariant to the *prestate* of some transitions, such as in `VotesOnce`, which makes sense to me.  Let's discuss at higher bandwidth.

Reply:
>  I think we will be applying the invariant to the prestate of some transitions, such as in `VotesOnce`, which makes sense to me. Let's discuss at higher bandwidth.

I agree that we should discuss at a higher bandwidth. The motivation for these changes is that I'm not sure that it makes sense to say that QCs in outputs came from the prestate and not the poststate. I will finish the changes I have in mind later today so we have something concrete to consider.